Cargo.toml

```diff
@@ -1,3 +1,23 @@
 [workspace]
 resolver = "2"
 members = ["tss-esapi", "tss-esapi-sys"]
+
+[patch.crates-io]
+p192 = { git = "https://github.com/RustCrypto/elliptic-curves.git" }
+p224 = { git = "https://github.com/RustCrypto/elliptic-curves.git" }
+sm2 = { git = "https://github.com/RustCrypto/elliptic-curves.git" }
+
+# https://github.com/RustCrypto/KDFs/pull/108
+concat-kdf = { git = "https://github.com/RustCrypto/KDFs.git" }
+
+cfb-mode = { git = "https://github.com/RustCrypto/block-modes.git" }
+
+# https://github.com/RustCrypto/RSA/pull/467
+rsa = { git = "https://github.com/baloo/RSA.git", branch = "baloo/oaep/non-string-label" }
+
+# https://github.com/RustCrypto/traits/issues/1738
+# https://github.com/RustCrypto/traits/pull/1742
+# Pending release of crypto-common 0.2.0-rc.2
+# Pending release of digest 0.11.0-rc.0
+crypto-common = { git = "https://github.com/RustCrypto/traits.git" }
+digest = { git = "https://github.com/RustCrypto/traits.git" }
```

Comment:
Are we waiting for these to be released on crates.io?

Reply:
Yeah, I'm relying on a bunch of pre-releases crates here. The kbkdf we just finished and released the first version of, but that relies on a bunch of pre-release or rc crates.

I've requested a new feature in the trait system for like detecting weak keys and be in compliance with the spec, probably safe to ignore that for now (and not merge this PR) until we have stable release.

The last huge pain point in the release cycle is a major rewrite of RSA, but I expect it to merge soon, and for the whole ecosystem to bump up.